template-parts/entry-author-bio.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * The template for displaying Author info
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty
+ * @since 1.0.0
+ */
+
+if ( (bool) get_the_author_meta( 'description' ) ) : ?>
+<div class="author-bio">
+		<h2 class="author-title heading-size-4">
+			<div class="author-avatar vcard">
+				<?php echo get_avatar( get_the_author_meta( 'ID' ), 160 ); ?>
+			</div>
+			<span class="author-name">
+				<?php
+				printf(
+					/* Translators: %s: post author */
```

Comment:
This content "By ...." is not inside an attribute, replace esc_attr with esc_html.